tasks/pip.yml

```diff
@@ -1,3 +1,11 @@
+- name: Install virtualenv with pip
```

Comment:
@moonwolf-github & @ritwickdsouza thinking about this virtualenv issue, have you thought about specifying the python version contained in `openwisp2_python` here and removing the symbolic link later?